metricflow/dag/id_prefix.py

```diff
@@ -67,3 +67,7 @@ class IdPrefix(Enum):
     EXEC_PLAN_PREFIX = "ep"
 
     MF_DAG = "mfd"
+
+    TIME_SPINE_SOURCE = "spine"
```

Comment:
Why make this change? Wouldn't keeping it `time_spine_src` reduce snapshot thrash considerably?